src/parser/priest/discipline/normalizers/AtonementSuccessiveDamage.ts

```diff
@@ -1,17 +1,17 @@
-import { EventType } from 'parser/core/Events';
-import EventsNormalizer from "parser/core/EventsNormalizer";
+import { DamageEvent, EventType, HealEvent } from 'parser/core/Events';
+import EventsNormalizer from 'parser/core/EventsNormalizer';
 
-import isAtonement from "../modules/core/isAtonement";
-import { ATONEMENT_DAMAGE_SOURCES } from "../constants";
+import isAtonement from '../modules/core/isAtonement';
+import { ATONEMENT_DAMAGE_SOURCES } from '../constants';
 
 class AtonementSuccessiveDamage extends EventsNormalizer {
-  normalize(events) {
-    const fixedEvents = [];
-    const _damageEventIndexes = [];
+  normalize(events: Array<ModifiedDamageEvent | ModifiedHealEvent>) {
```

Comment:
Event<any>

Reply:
Again, this module uses things that exist in ModifiedDamageEvent | ModifiedHealEvent, but not in the base Event<any>. How should I go about typing that?

Reply:
I've added casting to every place that you made this request. Lemme know if I should go down a different route!